README.md

```diff
@@ -195,7 +206,41 @@ Vaultenv will make the following environment variables available:
  - `FOOBAR` with the contents of the `foobar` field of the secret at
    `production/third-party`.
 
-## Allowed characters
+### Behavior configuration
+
+Vaultenv supports loading behavior configuration from files (in addition to the
+CLI flags and environment variable lookups). Vaultenv currently looks for these
+files in the following places:
+
+ - `/etc/vaultenv.conf`
+ - `$HOME/.config/vaultenv/config.conf`
```

Comment:
Is `config.conf` the name that the standard requires? Otherwise I'd call it `vaultenv.conf`.

Reply:
No, not really, was just my preference. I'll go for the name change